tests/php/test-amp-style-sanitizer.php

```diff
@@ -2703,6 +2703,20 @@ public function test_prioritized_stylesheets( $html_generator, $assert ) {
 			$this->markTestSkipped( 'Requires WordPress 5.0.' );
 		}
 		global $wp_theme_directories; // Note that get_theme_roots() does not work, for some reason.
+
+		// @todo Remove once https://github.com/WordPress/gutenberg/pull/23104 is in a release.
+		// Temporary fixes an issue with PHP errors being thrown in Gutenberg v8.3.0 on PHP 7.4.
+		if ( ! (
+			current_theme_supports( 'editor-color-palette' ) &&
+			current_theme_supports( 'editor-gradient-presets' ) &&
+			current_theme_supports( 'editor-font-sizes' )
+		) ) {
+			$args = [];
+			add_theme_support( 'editor-color-palette', $args );
+			add_theme_support( 'editor-gradient-presets', $args );
+			add_theme_support( 'editor-font-sizes', $args );
+		}
```

Comment:
Unless I'm misunderstanding the purpose of this logic here, if one or two of them are already set, but with a bool for example, this will still fail.

The logic needs to work on them individually.
```suggestion
		$theme_features = [
			'editor-color-palette',
			'editor-gradient-presets',
			'editor-font-sizes',
		];

		foreach ( $theme_features as $theme_feature ) {
			if ( ! current_theme_supports( $theme_feature ) ) {
				add_theme_support( $theme_feature, [] );
			}
		}
```

Reply:
Addressed in https://github.com/ampproject/amp-wp/pull/4901/commits/92058c5c29f39bb34eeb23c80c695033f9fa68e6.